subt_ign/src/GameLogicPlugin.cc

```diff
@@ -1189,13 +1214,15 @@ void GameLogicPluginPrivate::Finish()
     this->logStream.flush();
     this->eventStream.flush();
```

Comment:
I think this flush call can be moved to `LogEvent` function

Reply:
Removed: 762e1dd713a9e608ed1e4b302deb8273df15fc1d